xeol.yaml

```diff
@@ -0,0 +1,70 @@
+package:
+  name: xeol
+  version: 0.10.4
+  epoch: 0
+  description: A scanner for end-of-life (EOL) software
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/xeol-io/xeol
+      tag: v${{package.version}}
+      expected-commit: 76ef5b862f988a162bd4c9cdda2c144a3178efc3
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/go-git/go-git/v5@v5.13.0
+        github.com/hashicorp/go-getter@v1.7.5
+        github.com/golang-jwt/jwt/v4@v4.5.1
+        golang.org/x/net@v0.33.0
+        github.com/mholt/archiver/v3@v3.5.2
+        github.com/opencontainers/runc@v1.1.14
+        github.com/notaryproject/notation@v1.3.0
+      replaces: github.com/mholt/archiver/v3=github.com/anchore/archiver/v3@v3.5.2
+
+  - uses: go/build
+    with:
+      packages: ./cmd/xeol/main.go
+      output: xeol
+      ldflags: |
+        -X main.version=${{package.version}}
+        -X main.gitCommit=$(git rev-parse HEAD)
+        -X main.buildDate=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+        -X main.gitDescription=$(git describe --dirty --always --tags)
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compat package for ${{package.name}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"
+          ln -sf /usr/bin/xeol "${{targets.contextdir}}"/xeol
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+      pipeline:
+        - runs: /xeol --version | grep ${{package.version}}
+
+update:
+  enabled: true
+  github:
+    identifier: xeol-io/xeol
+    strip-prefix: v
+
+test:
+  pipeline:
+    - name: ${{package.name}} version and help
+      runs: |
+        xeol --version | grep ${{package.version}}
+        xeol --help
+    - name: Run an EOL check on curl image
+      runs: |
+        xeol cgr.dev/chainguard/curl:latest
```

Comment:
Just a sanity check, can we ensure it prints something to stdout|stderr? Simple `grep` check would work here. (My concern was that tool might just exits with zero without writing anything)